contrib/juliac.jl

```diff
@@ -53,11 +61,27 @@ end
 isnothing(outname) && error("No output file specified")
 isnothing(file) && error("No input file specified")
 
+function get_rpath(; relative::Bool = false)
+    if relative
+        if Sys.isapple()
+            return "-Wl,-rpath,'@loader_path/julia/' -Wl,-rpath,'@loader_path/'"
+        elseif Sys.islinux()
```

Comment:
Seems like this should be `Sys.isunix()` instead of `Sys.islinux()`? It's not Linux-specific AFAICT.